test/sequential/feeLock.API.feePayment.sequential.test.ts

```diff
@@ -114,7 +114,7 @@ test("gasless- GIVEN a feeLock configured (only Time and Amount )  WHEN the user
     firstCurrency,
     GASP_ASSET_ID,
     thresholdValue.sub(new BN(100)),
-    feeLockErrors.FeeLockingFail,
+    feeLockErrors.AccountBalanceFail,
```

Comment:
is this right? I think we need to be able to trigger the same error, did you investigate?

Reply:
Before that, we could pay fees for transactions in the first Currency tokens, but now it's available only in GASP, right? I think this is the reason why the error changed.

Reply:
We can pay fees using ETH too ( same as TUR or KSM  on previous tests)